index.html

```diff
@@ -32,16 +32,15 @@
     </head>
     <body>
         <header class="Header">
-            <div class="Header-icon">
-                <a href="http://walthamstuff.com"><img class="Header-iconImage" src="images/w.jpg" alt=""></a>
+           <div class="Header-icon">
+                <a href=""><img class="Header-iconImage" src="images/w.jpg" alt="Red letter W with yellow outline"></a>
```

Comment:
Maybe link the href here to simply `/`?

Also the alt text here wouldn't help me much if I was using a screen reader - you're right that describing images is important, and this would be the correct description for a photo, but in this case I feel something like `"Walthamstuff logo"` would be more appropriate to assist understanding of what the image represents...?

Code suggestion for ease of committing if you agree:
```suggestion
                <a href="/"><img class="Header-iconImage" src="images/w.jpg" alt="Walthamstuff logo"></a>
```